README.md

```diff
@@ -1 +1,21 @@
-# Sport
\ No newline at end of file
+# Football_teams
+
+### About :
+
+our website allows user to search about footbal teams in specific country ,displays the teams in this country, and some details about it.
```

Comment:
A few small grammatical mistakes: 'football', comma in the wrong place ',displays'